include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -824,7 +824,7 @@ class CommonOperatorLoader : public ProtobufLoader {
     return Error::success();
   }
 
-  Error loadSparseLengthsWeightedSumOffsets(const OpType &op) {
+  Error loadEmbeddingBag(const OpType &op) {
```

Comment:
Previous naming is kind of OK though.

Reply:
Changed since there's no longer 'lengths' lol..